chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java

```diff
@@ -90,7 +90,10 @@ public void writeModelsToDisk() {
         chunker.save();
         System.out.println("Done training, models are in " + rm.getString("modelDirPath"));
     }
-
+    public void writeModelsToDisk(String dir, String modelName){
+        chunker.write(dir + modelName + ".lc", dir + modelName + ".lex");
```

Comment:
Minor: you can be safe and make sure there is [already a folder. ](https://github.com/IllinoisCogComp/illinois-cogcomp-nlp/blob/master/core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/io/IOUtils.java#L58)